components/eamxx/src/dynamics/homme/physics_dynamics_remapper.cpp

```diff
@@ -426,6 +426,10 @@ do_remap_fwd()
   const int team_size = std::min(256, std::min(128*m_num_phys_cols,32*(concurrency/this->m_num_fields+31)/32));
 #endif
 
+#ifdef KOKKOS_ENABLE_SYCL
+  const int team_size = 4;
```

Comment:
I am puzzled by this. Do we really want just 4 on sycl? It's staggeringly different from other GPU platforms' default...

Reply:
the change is from Nov 2023. i looked at my notes and did not see why it was needed. i will try to remove it.

Reply:
This is for the P-D remapper (not the GllFvPhys one), so its relevance is somewhat limited (doesn't run at runtime, usually). So not a huge deal. But if things work without, no point in limiting team size to 4.

Reply:
used hip TS calculation and it worked at least for ne4 run.